File: comms/push-notifications/users/list-users/list-users.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\PushNotifications\Users\Requests\UsersListRequest;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->pushNotifications->users->list(
    new UsersListRequest([
        'pageToken' => 'pageToken',
        'pageSize' => 50,
    ]),
);

File: comms/push-notifications/users/list-users/list-users.cs
using Twilio.Comms;
using System.Threading.Tasks;
using Twilio.Comms.PushNotifications;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "TWILIO_ACCOUNT_SID",
            authToken: "TWILIO_AUTH_TOKEN"
        );

        await client.PushNotifications.Users.ListAsync(
            new UsersListRequest {
                PageToken = "pageToken",
                PageSize = 50
            }
        );
    }

}
